centaur/src/main/resources/standardTestCases/papi_v2_gcsa/papi_v2_gcsa.wdl

```diff
@@ -0,0 +1,30 @@
+version 1.0
+
+workflow papi_v2_gcsa {
+    call get_token_info
+    output {
+        String email = get_token_info.json.email
+        String scope = get_token_info.json.scope
+    }
+}
+
+# Confirm that even though the SA is being used by the backend to create the pipeline job,
+# the GCSA is the actual account used to run the container
```

Comment:
It might be nice to mention the meaning of GCSA in this comment

Google helpfully suggests _Golf Course Superintendents Association_ but I don't think that's what you have in mind 😄